deploy.sh

```diff
@@ -1,12 +1,16 @@
 #!/bin/bash
 
-echo "🚀 : Deploying roboflow/inference-server:cpu"
-docker push "roboflow/inference-server:cpu"
+# needed to build multi-platform images
+docker buildx create --name roboflow
+docker buildx use roboflow
+
+echo "🛠 : Building roboflow/inference-server:cpu"
+docker buildx build --platform linux/amd64,linux/arm64 --push -t "roboflow/inference-server:cpu" -f- . < Dockerfile.cpu
 
 echo ""
-echo "🚀 : Deploying roboflow/inference-server:gpu"
-docker push "roboflow/inference-server:gpu"
+echo "🛠 : Building roboflow/inference-server:gpu"
+docker buildx build --platform linux/amd64 --push -t "roboflow/inference-server:gpu" -f- . < Dockerfile.gpu
 
 echo ""
-echo "🚀 : Deploying roboflow/inference-server:jetson"
-docker push "roboflow/inference-server:jetson"
+echo "🛠 : Building roboflow/inference-server:jetson"
+docker buildx build --platform linux/arm64 --push -t "roboflow/inference-server:jetson" -f- . < Dockerfile.jetson
```

Comment:
Didn't know you could build and push in one fell swoop, nice

Reply:
I think that might be part of buildx